index.bs

```diff
@@ -990,6 +990,174 @@ while (true) {
 }
 </pre>
 
+## Complete example ##  {#example-complete}
+
+*This section is non-normative.*
+
+This example illustrates use of the closed and ready promises, opening
+of uni-directional and bi-directional streams by either the client or
+the server, and sending and receiving datagrams.
+
+<pre class="example" highlight="js">
+// Adds an entry to the event log on the page, optionally applying a specified
+// CSS class.
+function addToEventLog(text, severity = 'info') {
```

Comment:
Since this utility function isn't related to WebTransport, it might be an unnecessary distraction. Perhaps leave it as an exercise for the reader instead?

Reply:
Moved it to the end, so it will be less distracting.